.travis.yml

```diff
@@ -6,11 +7,12 @@ before_install:
   - echo "options(repos = c(CRAN='http://cran.r-project.org'))" > .Rprofile
 
 before_script:
-  - Rscript -e "install.packages('knitr', repos='http://cran.cnr.Berkeley.edu/')"
-  - Rscript -e "install.packages('rmarkdown', repos='http://cran.cnr.Berkeley.edu/')"
+  - Rscript -e "install.packages('knitr')"
+  - Rscript -e "install.packages('rmarkdown')"
 
 r_github_packages:
   - jimhester/covr
+  - hadley/testthat
```

Comment:
Interesting. Was this necessary? testthat should be automatically installed from CRAN in the Travis run because it's included in the Suggests field of the DESCRIPTION (https://github.com/McDowellLab/loadflex/blob/master/DESCRIPTION#L33). This line will cause the GitHub version to be installed, but I'm not sure we need that (and not sure whether it happens before or after the CRAN installs, anyway). What was your reasoning for adding this?